src/components/common/Slider/Slider.tsx

```diff
@@ -0,0 +1,58 @@
+import Slider from '@material-ui/core/Slider'
+import styled from 'styled-components/macro'
+
+interface SliderProps {
+  title: string
+  startValue: number
+  min: number
+  max: number
+  step: number
+  onChange: Function
+}
+
+const SliderComponent = ({
+  title,
+  startValue,
+  min,
+  max,
+  step,
+  onChange,
+}: SliderProps) => {
+  return (
+    <label>
+      <CriteriaName>{title}</CriteriaName>
+      <StyledSlider
+        defaultValue={startValue}
+        min={min}
+        max={max}
+        step={step}
+        onChange={(event: React.ChangeEvent<{}>, value: number | number[]) =>
+          onChange(value as number)
+        }
+        // aria-labelledby="range-slider"
+      />
+    </label>
+  )
+}
+
+export default SliderComponent
+
+const CriteriaName = styled.span`
+  font-size: 1.5rem;
+  font-weight: 400;
+  letter-spacing: 2px;
```

Comment:
why you use 2px here and not 0.12rem? (if your font-size is 16px this would be around 2px)